src/CarbonAware.WebApi/src/Models/EmissionsForecastDTO.cs

```diff
@@ -96,7 +95,7 @@ public record EmissionsForecastDTO
     [JsonPropertyName("forecastData")]
     public IEnumerable<EmissionsDataDTO>? ForecastData { get; set; }
 
-    public static EmissionsForecastDTO FromEmissionsForecast(EmissionsForecast emissionsForecast)
+    public static EmissionsForecastDTO FromEmissionsForecast(global::CarbonAware.Model.EmissionsForecast emissionsForecast)
```

Comment:
Seems like this method is not longer needed.

Reply:
Removed